apps/hash-external-services/README.md

```diff
@@ -12,3 +12,34 @@ These will be migrated into semantic folders in the future. For example, within
 1.  `hash-external-services/opensearch` → `hash-search`
 1.  `hash-external-services/postgres` → `hash-database`
 1.  `hash-external-services/temporal` → `hash-executor`
+
+## Database Backups
+
+It is possible to set up automated PostgreSQL backups using an additional service. You can add this service directly to the desired `docker-compose` or add another `docker-compose` as a layered config.
+
+You can add the following service:
+
+```yaml
+postgres-backup:
+  image: kartoza/pg-backup:15-3.3
+  depends_on:
+    postgres:
+      condition: service_healthy
+  environment:
+    POSTGRES_USER: "${POSTGRES_USER}"
+    POSTGRES_PASS: "${POSTGRES_PASSWORD}"
+    POSTGRES_HOST: postgres
+    POSTGRES_PORT: 5432
+    # Change the name of the dumps here
+    ARCHIVE_FILENAME: "$$(date +'%Y-%m-%dT%H-%M-%S').gz"
+    DUMP_ARGS: -Fp -Z9
```

Comment:
maybe link to the article we used for this, otherwise it's a bit opaque

Reply:
(https://www.postgresql.org/docs/current/app-pgdump.html)

Reply:
Oh yes, thank you! e55a4ace4e99e06f95be6922bc674b791d0cbfc3